# packaging of opencv DMatch object https://docs.opencv.org/3.4/d4/de0/classcv_1_1DMatch.html
# this is meant to fullfill the constructor to make a DMatch
# this message is only defined to define other messages, not to be used directly

int32 		QueryIdx
int32 		TrainIdx
int32 		ImgIdx
float64 	Distance 


